compiler/test/dotc/neg-init-global-scala2-library-tasty.excludelist

```diff
@@ -1,9 +1,6 @@
 ## See #18882
 patmat.scala
 t9312.scala
-unapplySeq-implicit-arg.scala
-unapplySeq-implicit-arg2.scala
-unapplySeq-implicit-arg3.scala
```

Comment:
Our goal is to move all tests shown here to warn-tasty since they rely on tasty, but they also trigger some warnings in scala2 library which is hard to document